src/components/landing/OfferingCards.js

```diff
@@ -0,0 +1,62 @@
+import OfferingCard from "./OfferingCard";
+import { FaUsers } from "react-icons/fa6";
+import { RiPresentationFill } from "react-icons/ri";
+import { GiPodium } from "react-icons/gi";
+import { LiaNewspaperSolid } from "react-icons/lia";
+
+const Offerings = [
+  {
+    title: "SEMINARS",
+    description:
+      "In-depth seminars offering insight into AI's latest technologies and advancements.",
+    icon: <FaUsers className="size-12 text-[#4F9AD7]" />,
+  },
+  {
+    title: "GUEST SPEAKERS",
+    description:
+      "Engaging sessions with AI experts, industry professionals, and prominent researchers.",
+    icon: <RiPresentationFill className="size-12 text-[#4F9AD7]" />,
+  },
+  null,
```

Comment:
remove these nulls, i know the design was weird but may look better without them